shared/chat/conversation/messages/wrapper/shared.js

```diff
@@ -94,88 +94,99 @@ const Failure = ({failureDescription, onEdit, onRetry, onCancel}) => {
   )
 }
 
+const LeftSide = props => (
```

Comment:
The avatar overlaps the send indicator:
<img width="99" alt="image" src="https://user-images.githubusercontent.com/11968340/40322656-10b54e9e-5d01-11e8-9cca-2c7dbceb87ad.png">

Reply:
moved to the right side